src/IW/Server.hs

```diff
@@ -3,15 +3,23 @@ module IW.Server
        , server
        ) where
 
+import Servant.API ((:<|>))
 import Servant.API.Generic (toServant)
 import Servant.Server (Server, hoistServer)
 
 import IW.App (AppEnv)
 import IW.Effects.Log (runAppAsHandler)
-import IW.Server.Issue (IssueAPI, issueServer)
+import IW.Server.Issue (IssuesAPI, issuesHandler)
+import IW.Server.Repo (ReposAPI, reposHandler)
 
 
-type API = IssueAPI
+type API = ReposAPI :<|> IssuesAPI
```

Comment:
I thinл that sticking to the same scheme and conventions would make sense here as well. Instead, you can do something like this and it would be much more consistent and structured:

```haskell
data IwSite route = IwSite
    { iwIssuesRoute :: route :- IssuesApi
    , iwIReposRoute :: route :- ReposApi
    } deriving (Generic)

type IwApi = ToApi IwSite

iwServer :: IwSite AppServer
iwServer = IwSite
    { iwIssuesRoute = toServant ...
    , iwReposRoute = toServant ...
    }
```

and also, in this case you don't need to change `Server.Issue` and `Server.Repo` servers and use them instead of handlers here.

Reply:
@vrom911 Got it.

Reply:
@vrom911 Yeah this makes sense, so do you ever use the `(:<|>)` operator when using `servant`?

Reply:
We don't use it anywhere, to be honest. `servant-generic` (which is currently merged to `servant` itself and which we are using everywhere) technique is much nicer to my taste. But it's possible to use for some small stuff.